lib/active_admin/resource/belongs_to.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-require_relative "../resource"
```

Comment:
is it possible that this is a breaking change? Like some use cases where `resource/belongs_to` is required individually without other resources?

Reply:
I don't know. If the warning appears in normal usage of ActiveAdmin v4 and v3 (meaning a real app, not the test/dev app we generate for the project), we should address it in both versions. If we can see it stable in a real app on v3 then I think it's safe.

Reply:
In commit https://github.com/activeadmin/activeadmin/commit/79c1a980850275c91af1bbe3570062ad8ffa7a25 is where the circular require is introduced but doesn't seem to have any need for it so this removal should be safe. We can backport it as well.